recipes/itk/all/conanfile.py

```diff
@@ -223,8 +224,17 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, self._cmake_module_dir, "Modules"))
+        # Do not remove UseITK.cmake and *.h.in files
+        for cmake_files in glob.glob(os.path.join(self.package_folder, self._cmake_module_dir, "*.cmake")):
```

Comment:
As far as I can see this will only keep the `UseITK.cmake`-file - every other file is removed.
My original idea was to just copy the `UseITK.cmake` into a new directors (e.g. `build_modules`) before `cmake.install()` and then remove all directory as it was done in the original script:

`
self.copy("UseITK.cmake", dst = "build_modules", src = os.path.join(self._source_subfolder, "CMake"))
`
No need for `_cmake_module_dir` or `glob`. Am I missing something?

Reply:
`UseITK.cmake` defines several CMake functions requiring those `.h.in` files.

Reply:
You're absolutely right, I overlooked the `.h.in` files.